app/services/iaa_reporting_helper.rb

```diff
@@ -17,6 +17,15 @@ def key
     end
   end
 
+  PartnerConfig = Struct.new(
+    :partner_account_id,
+    :partner_agency,
+    :issuer,
```

Comment:
A partner can have multiple issuers, so this should be plural, right? The goal of this helper is to aggregate issuers by partner account
```suggestion
    :issuers,
```